go/updater/updater.go

```diff
@@ -339,7 +365,14 @@ func (u *Updater) update(ui libkb.UpdateUI, force bool, requested bool) (update
 
 	_, err = u.restart(ui)
```

Comment:
The update quit response, you can pass back quit: false, which means the client doesn't want to be restarted:
https://github.com/keybase/client/blob/master/protocol/avdl/update_ui.avdl#L24

So the Electron app should return false for linux:
https://github.com/keybase/client/blob/master/react-native/react/actions/update.js#L94

Which it currently does since it's only true for osx (if app bundle).